### PR TITLE
Add export CSV button

### PR DESCRIPTION
**Pull request by ollytheninja**
This is duplicating a bit of code from the items endpoint, but seemed like the best way to do it.

### CODE REVIEW

**Comment by monkeysecurity** [comment]
If any of the fields contain a comma, I don't see anything that would escape it before exporting.

Do we not have any issues or notes that contain a comma?

**Comment by cstewart87** [comment]
We ran into some issues trying to cherry-pick this feature.  Has anyone else tested yet?  Also, is there an ETA for when this may be merged into the project?

**Comment by monkeysecurity** [comment]
Hey @cstewart87,

From a code review, I was concerned that it wouldn't handle commas in issue notes correctly.  I haven't tested it, but I was wanting to test/merge it as soon as that concern was met.

Do you think it's good to go?

**Comment by cstewart87** [comment]
@monkeysecurity - this isn't an immediate priority for us right now, but I'll see if we can't test a bit more soon and provide some feedback.

File: security_monkey/export/__init__.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+    out = ",".join(["/".join(at) for at in attributes]) + "\n"
+
+    for issue in query:
+        values = []
+        for attribute in attributes:
+            val = issue
+            for at in attribute:
+                val = getattr(val, at)
+            values.append('"'+str(val)+'"')
```
Comment (monkeysecurity) [inline security_monkey/export/__init__.py]: http://stackoverflow.com/questions/769621/dealing-with-commas-in-a-csv-file

The double quotes you're adding will help if the value has a comma, or a CRLF, but not if the value has a double quote:

https://tools.ietf.org/html/rfc4180

> (6.) Fields containing line breaks (CRLF), double quotes, and commas
>        should be enclosed in double-quotes.  For example:
> 
>        "aaa","b CRLF
>        bb","ccc" CRLF
>        zzz,yyy,xxx
> 
>
> (7.)  If double-quotes are used to enclose fields, then a double-quote
>       appearing inside a field must be escaped by preceding it with
>       another double quote.  For example:
>
>       "aaa","b""bb","ccc"

So I'm going to add a line after 156 replacing any `"` in val with `""`

**Comment by monkeysecurity** [comment]
Closing as this is a duplicate of #390, which has been merged.

**Pull request closed without merging**